proof_test.go

```diff
@@ -13,6 +13,76 @@ import (
 	"github.com/celestiaorg/nmt/namespace"
 )
 
+// TestVerifyNamespace_EmptyProof tests the correct behaviour of VerifyNamespace for valid and invalid empty proofs.
+func TestVerifyNamespace_EmptyProof(t *testing.T) {
+	// create a tree with 4 leaves
+	nIDSize := 1
+	tree := exampleNMT(nIDSize, 1, 2, 3, 4)
+	root, err := tree.Root()
+	require.NoError(t, err)
+
+	// build a proof for an NID that is outside tree range of the tree
```

Comment:
"outside tree range of the tree" seems slightly wrong but I'm not sure what it meant to say. Perhaps

```suggestion
	// build a proof for an NID that is outside the range of the tree
```

Reply:
Thanks, addressed https://github.com/celestiaorg/nmt/pull/180/commits/8a276366b706a678c296a10aa718fc1d0667fb14